.rubocop.yml

```diff
@@ -1,5 +1,6 @@
 require:
   - rubocop-rails
+  - rubocop-minitest
```

Comment:
What stuff does this bring in?

Reply:
Adds some additional Cops for minitest-specific things.